### ISSUE TITLE
`derived` and `$derived` cause lsp error

### ISSUE THREAD

**Issue by Pandoks** [sveltejs/language-tools#2605]
### Describe the bug

When you import `derived` from `svelte/store`, it seems like `$derived` breaks the lsp.
![image](https://github.com/user-attachments/assets/ae79e890-7e4d-4c22-b6bc-c1b38372a501)

### Reproduction

Use `derived` from `svelte/store` in the same file as `$derived`

### System Info

```shell
System:
    OS: macOS 15.1.1
    CPU: (10) arm64 Apple M1 Max
    Memory: 13.46 GB / 64.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.5.0 - ~/.nvm/versions/node/v20.5.0/bin/node
    npm: 10.3.0 - ~/.nvm/versions/node/v20.5.0/bin/npm
    pnpm: 9.9.0 - ~/.nvm/versions/node/v20.5.0/bin/pnpm
    bun: 1.1.36 - /opt/homebrew/bin/bun
    Watchman: 2024.11.18.00 - /opt/homebrew/bin/watchman
  Browsers:
    Brave Browser: 131.1.73.91
    Safari: 18.1.1
  npmPackages:
    svelte: ^5.2.7 => 5.2.7
```

### Severity

annoyance

**Comment by Pandoks** [comment]
> Yeah but you should be able to synchronise stores with `$derived`

oh i didn't even know that. there should be more info on migrating stores to runes. i know it's technically not a fully deprecated feature, but there isn't many examples of using stores together with runes.  i thought if you used stores, you have to strictly use the stores api

**Comment by nmzein** [comment]
Yeah should prob be better documented but stores are fully compatible with the new runes stuff so you can use them in all runes including state, derived and effect. This is useful because it makes things like dealing with SvelteKit stuff that hasnt been migrated to runes yet easier without having to use deprecated apis

**Comment by jasonlyu123** [comment]
Fixed in language-server 0.17.16 and svelte-check 4.2.2
